mockup/patterns/thememapper/pattern.thememapper.less

```diff
@@ -1,6 +1,6 @@
 @import "@{mockuplessPath}/ui.less";
 /* just be lazy here and include all of bootstrap... */
-@import "@{mockupPath}/filemanager/pattern.filemanager.less";
+@import "@{mockup-patterns-filemanager}";
```

Comment:
I just realised that this is a separate branch, I context switch and scanned through this. Yes, I agree.